block_course_contents.php

```diff
@@ -164,15 +164,41 @@ public function get_content() {
                 $title = $format->get_section_name($section);
             }
 
+            if ( ($i == 0) && (!empty($globalconfig->display_course_link)) ) {
```

Comment:
There seems to be some extra whitespace next to outer brackets.

Reply:
I added that for clarity, but I'm happy to correct in the next pull request.

Reply:
Yeah but Moodle has its coding style which we follow in plugins, too. Please see https://docs.moodle.org/dev/Coding_style